ext4: remove double definitions of xattr macros

From: Shen Feng <shen@cn.fujitsu.com>

remove the definitions of macros:
XATTR_TRUSTED_PREFIX
XATTR_USER_PREFIX
since they are defined in linux/xattr.h

Signed-off-by: Shen Feng <shen@cn.fujitsu.com>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>
---
 fs/ext4/xattr_trusted.c |    4 +---
 fs/ext4/xattr_user.c    |    4 +---
 2 files changed, 2 insertions(+), 6 deletions(-)

Index: linux-2.6.26-rc5/fs/ext4/xattr_trusted.c
===================================================================
--- linux-2.6.26-rc5.orig/fs/ext4/xattr_trusted.c	2008-06-06 17:03:38.000000000 -0700
+++ linux-2.6.26-rc5/fs/ext4/xattr_trusted.c	2008-06-06 17:03:45.000000000 -0700
@@ -13,13 +13,11 @@
 #include "ext4.h"
 #include "xattr.h"
 
-#define XATTR_TRUSTED_PREFIX "trusted."
-
 static size_t
 ext4_xattr_trusted_list(struct inode *inode, char *list, size_t list_size,
 			const char *name, size_t name_len)
 {
-	const size_t prefix_len = sizeof(XATTR_TRUSTED_PREFIX)-1;
+	const size_t prefix_len = XATTR_TRUSTED_PREFIX_LEN;
 	const size_t total_len = prefix_len + name_len + 1;
 
 	if (!capable(CAP_SYS_ADMIN))
Index: linux-2.6.26-rc5/fs/ext4/xattr_user.c
===================================================================
--- linux-2.6.26-rc5.orig/fs/ext4/xattr_user.c	2008-06-06 17:03:38.000000000 -0700
+++ linux-2.6.26-rc5/fs/ext4/xattr_user.c	2008-06-06 17:03:45.000000000 -0700
@@ -12,13 +12,11 @@
 #include "ext4.h"
 #include "xattr.h"
 
-#define XATTR_USER_PREFIX "user."
-
 static size_t
 ext4_xattr_user_list(struct inode *inode, char *list, size_t list_size,
 		     const char *name, size_t name_len)
 {
-	const size_t prefix_len = sizeof(XATTR_USER_PREFIX)-1;
+	const size_t prefix_len = XATTR_USER_PREFIX_LEN;
 	const size_t total_len = prefix_len + name_len + 1;
 
 	if (!test_opt(inode->i_sb, XATTR_USER))
